packages/strapi-hook-bookshelf/lib/index.js

```diff
@@ -470,6 +470,7 @@ module.exports = function(strapi) {
                       try {
                         const connection = strapi.config.connections[definition.connection];
                         let columns = Object.keys(attributes).filter(attribute => ['string', 'text'].includes(attributes[attribute].type));
+                        let indexes;
```

Comment:
Why? It seems that the variable isn't used anywhere. We should keep it in the scope of the `if` condition?